### ISSUE TITLE
Zip code error when creating new account

### ISSUE THREAD

**Issue by ythuang123** [jkhartshorne/KidTalkIssues#39]
I tried to create a new account on the staging site and got this wacky error. It doesn't seem to matter what zip code I put in or whether I leave it blank. This error always popped up. I'm using: iPhone SE (version iOS 13.5.1) on Safari (latest?).

![2020-08-13_21-18-23_000](https://user-images.githubusercontent.com/68764650/90202555-0ec31200-ddac-11ea-9113-db71b4b7eaed.png)

**Comment by alexchippendale** [comment]
@ythuang123 Hmm that's interesting, that error typically shows up when there are an incorrect number of characters for a 5 digit zip code or if there are strange characters in the field. What zip code did you put into it? Also check that there aren't a bunch of blank spaces or blank spaces followed by extra character. You can also try going all the way to the right of the text in that filed, deleting everything, and then typing it in again.

**Comment by ythuang123** [comment]
@alexchippendale -- eh, the internet is a funny place, amirite? ;) I'm 100% sure that I put in legit zip codes and made sure I got rid of the blank spaces (come on Alex, rookie mistakes). But I logged in and out again today and was able to put in those same numbers, blanks, and didn't get the error. So I'm closing this out. I'll re-open it if it appear again. (although just to make me feel like it was worth my time to discover this, can we fix the typo in the error message --> "there was and issue..." should be "there was an issue..." Thanks for your help!
